proto/control.proto

```diff
@@ -6,6 +6,7 @@ service PDPControl {
   rpc Request (Item) returns (Response) {}
   rpc Upload (stream Chunk) returns (Response) {}
   rpc Apply (Update) returns (Response) {}
+  rpc NotifyReady (Empty) returns (Response) {}
```

Comment:
Don't gRPC allow method with no arguments?

Reply:
With empty brackets there will be an error. Thus we do have an argument: Empty (it is a message).